data_steward/cdr_cleaner/manual_cleaning_rules/dc732_suppress.py

```diff
@@ -0,0 +1,598 @@
+        backup_rows_dest_table = get_backup_table_for(table_name)
+        job_config = bigquery.QueryJobConfig(destination=backup_rows_dest_table)
+        job_config.write_disposition = 'WRITE_APPEND'
+        backup_table_rows_query = BACKUP_TABLE_ROWS_QUERY.render(
+            dataset_ids=dataset_ids,
+            table=table_name,
+            suppress_rows_table=suppress_rows_table)
+        backup_table_query_job = CLIENT.query(backup_table_rows_query,
+                                              job_config=job_config)
+        query_jobs.append(backup_table_query_job)
+
+        if ext_table_name:
+            backup_ext_dest_table = get_backup_table_for(ext_table_name)
+            job_config = bigquery.QueryJobConfig(
+                destination=backup_ext_dest_table)
+            job_config.write_disposition = 'WRITE_APPEND'
+            backup_ext_table_rows_query = BACKUP_EXT_TABLE_ROWS_QUERY.render(
+                dataset_ids=dataset_ids,
+                table=table_name,
+                suppress_rows_table=suppress_rows_table)
+            backup_ext_table_query_job = CLIENT.query(
+                backup_ext_table_rows_query, job_config=job_config)
+            query_jobs.append(backup_ext_table_query_job)
+    return query_jobs
+
+
+# -
+
+# # Execute Deletion
+# Use a meta-query to generate the minimal set of DELETE statements needed to delete rows in
+# domain and \_ext tables.
+
+# +
+GET_DELETE_QUERIES = JINJA_ENV.from_string("""
+WITH table_ids AS
+(SELECT DISTINCT dataset_id, table 
+FROM `{{suppress_rows_table}}`)
+SELECT CONCAT('DELETE FROM `', 
+ dataset_id, '.', table, '` t', 
+ ' WHERE EXISTS', 
+ '(SELECT 1 FROM {{suppress_rows_table}} WHERE dataset_id = "', 
+ dataset_id, '" AND table = "', table, '" AND t.', table, '_id = row_id)')
+AS query_string
+FROM table_ids
+""")
+
+
+def to_ext_query(delete_query):
+    """
+    From a statement which deletes from a domain table, derive a statement
+    which deletes from the corresponding extension table
+
+    :param delete_query: a statement which deletes from a domain table
+    :return: statement which deletes from an extension table
+    """
+    return delete_query.replace('` t WHERE', '_ext` t WHERE')
+
+
+def get_delete_queries(suppress_rows_table):
+    """
+    Get list of DELETE statements needed to delete rows in domain and _ext tables
+    
+    :param suppress_rows_table: identifies the table where suppress rows are loaded
+    """
+    get_delete_query = GET_DELETE_QUERIES.render(
+        suppress_rows_table=suppress_rows_table)
+    delete_query_job = CLIENT.query(get_delete_query)
+    delete_query_results = list(delete_query_job.result())
+    all_delete_queries = []
+    for delete_query_result in delete_query_results:
+        delete_query = delete_query_result['query_string']
+        delete_ext_query = to_ext_query(delete_query)
+        all_delete_queries.extend([delete_query, delete_ext_query])
+    return all_delete_queries
+
+
+# -
```

Comment:
nit:  unnecessary comment.

Reply:
This is a delimiter used by jupytext to distinguish cells.